openquake/server/views.py

```diff
@@ -1510,7 +1528,9 @@ def download_aggrisk(request, calc_id):
         },
     )
     writer = csv.writer(response)
-    writer.writerow(losses.dtype.names)
+    headers = [FIELD_DESCRIPTION[field] if field in FIELD_DESCRIPTION else field
+               for field in losses.dtype.names]
```

Comment:
I don't like this.

Reply:
I am keeping the csv as it was, for consistency with all the other csv files. In the html table on top of the outputs page, I am displaying both the original names and the human-readable names.